contracts/MarketCollateralised.sol

```diff
@@ -89,21 +106,22 @@ abstract contract MarketCollateralised is Market {
 					_self,
 					result
 				);
-				
-				emit Borrowed(index, result);
 
+				// Add the amount we borrowed to the total collateral				
+				emit Borrowed(index, result);
 				_totalCollateral += result;
 			}
+			// Add the amount we borrowed to the race collateral
 			_marketCollateral[marketId] += result;
 		}
-
+		// Store the amount of exposure that this bet added. This will be deducted from the total exposure when the bet is paid out
 		_betExposure[_bets.length] = result;
-		return result;
+		return result; // To be added to total exposure
```

Comment:
just double checking that it _does_ get added and the comment isn't a TODO?

Reply:
It is just a tip, letting you know what will happen with the result. Sounds like a TODO so I'll change the wording to make it clearer